tornjak-frontend/src/components/entry-create-json.js

```diff
@@ -0,0 +1,825 @@
+
+                                                    }
+                                                    {index === 0 && this.state.selectedEntryId !== 0 &&
+                                                        <div>
+                                                            <Link
+                                                                id="firstEntry"
+                                                                href="#"
+                                                                renderIcon={NextOutline16}
+                                                                visited={false}
+                                                                onClick={(e) => {
+                                                                    this.setSelectedEntriesIds(entryId, index);
+                                                                    e.preventDefault();
+                                                                }}
+                                                            >
+                                                                {(index + 1).toString() + ". " + entryId.spiffeId}
+                                                            </Link>
+                                                        </div>
+
+                                                    }
+                                                    {index > 0 &&
+                                                        <div>
+                                                            <Link
+                                                                id={entryId.spiffeId}
+                                                                href="#"
+                                                                renderIcon={NextOutline16}
+                                                                visited={false}
+                                                                onClick={(e) => {
+                                                                    this.setSelectedEntriesIds(entryId, index);
+                                                                    e.preventDefault();
+                                                                }}
+                                                            >
+                                                                {(index + 1).toString() + ". " + entryId.spiffeId}
+                                                            </Link>
+                                                        </div>
+                                                    }
+                                                </div>
+                                            ))}
+                                        </fieldset>
+                                        <br></br>
+                                        <legend className="additional_info_entries_list">[Select Entry to Populate Metadata to Free Form]</legend>
```

Comment:
Can we change it to:
"Select Entry to Edit"

Reply:
done!